src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ActionPlanAppointmentRepositoryTest.kt

```diff
@@ -30,9 +30,9 @@ class ActionPlanAppointmentRepositoryTest @Autowired constructor(
   fun setup() {
     actionPlanAppointmentRepository.deleteAll()
     actionPlanRepository.deleteAll()
+    endOfServiceReportRepository.deleteAll()
```

Comment:
EOSR needs to be cleared before the referral repository due to a foreign key constraint